docs/resources/webhook.md

```diff
@@ -0,0 +1,97 @@
+---
+page_title: "Resource hcp_webhook - terraform-provider-hcp"
+subcategory: "Cloud Platform"
+description: |-
+  The webhook resource manages a HCP webhook, used to notify external systems about a project resource's lifecycle events
+---
+
+# hcp_webhook (Resource)
+
+The webhook resource manages a HCP webhook, used to notify external systems about a project resource's lifecycle events
+
+## Example Usage
+
+```terraform
+resource "hcp_webhook" "example" {
```

Comment:
What do you think about making the name of the resource slightly more specific. I was thinking something like `hcp_notifications_webhook`. The thought being that in the future we may have inbound web hooks as well to trigger actions. This change would potentially make it more clear what this resource is for and potentially prevent a name collision in the future.

Reply:
That's a good point. I like your suggestion. I am just going to run it with Jordan because he approved the design and just want to make sure he is aware of the change.